Jenkinsfile

```diff
@@ -12,22 +12,18 @@ node('CLIENT') {
         checkout scm
         sh '''git submodule update --init'''
 
-    stage 'Qt5'
-        sh '''rm -rf build
+    stage 'Linux'
+        def linux = docker.image('dominikschmidt/owncloud-client-ci-image:latest')
+        linux.pull()
+        linux.inside {
+            sh '''
 		mkdir build
 		cd build
-		cmake -DCMAKE_BUILD_TYPE="Debug" -DUNIT_TESTING=1 -DWITH_TESTING=1 -DCMAKE_PREFIX_PATH=/var/lib/jenkins/qt/5.6.2 ..
+		cmake -DCMAKE_BUILD_TYPE="Debug" -DUNIT_TESTING=1 ..
 		make -j4
-		ctest -V --output-on-failure'''
-
-    stage 'Qt5 - clang'
```

Comment:
you are dropping the clang compilation ... intended?

Reply:
I took over a diff @ogoffart gave me, was about to discuss/readd this once the normal GCC compilation works.

Reply:
We can have the discussion here actually:
@ogoffart was that intended or just for quicker testing? You installed clang in the docker image after all.

Reply:
in the past we had to compile with gcc and clang just because in some situations one of them was failing and the code needed to be adjusted